### DISCUSSION TITLE
2.0.0-beta.6

### DISCUSSION THREAD

**Discussion by shuding** [vercel/swr#2062] [New]
## What's Changed
* fix: Fix typos by @hiro0218 in https://github.com/vercel/swr/pull/2052
* feat: Add DevTools hook by @koba04 in https://github.com/vercel/swr/pull/2016
* fix: Use consistent typing imports by @huozhi in https://github.com/vercel/swr/pull/2061

## New Contributors
* @hiro0218 made their first contribution in https://github.com/vercel/swr/pull/2052

**Full Changelog**: https://github.com/vercel/swr/compare/2.0.0-beta.5...2.0.0-beta.6

<hr /><em>This discussion was created from the release <a href='https://github.com/vercel/swr/releases/tag/2.0.0-beta.6'>2.0.0-beta.6</a>.</em>

**Comment by StringKe** [comment]
Is there any documentation available for version 2.0?

**Comment by rodbs** [comment]
Any updates about final release date? thx
